rand_distr/src/poisson.rs

```diff
@@ -232,6 +240,13 @@ where
     }
 }
 
+impl Distribution<u64> for Poisson<f64> {
+    #[inline]
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> u64 {
+        <Poisson<f64> as Distribution<f64>>::sample(self, rng) as u64
```

Comment:
Acceptable since float-to-int casts for too-large values will saturate to the maximum supported type since Rust 1.45. We require a more recent Rust version anyway so this is fine.